Include/longobject.h

```diff
@@ -26,6 +26,10 @@ PyAPI_FUNC(size_t) PyLong_AsSize_t(PyObject *);
 PyAPI_FUNC(unsigned long) PyLong_AsUnsignedLong(PyObject *);
 PyAPI_FUNC(unsigned long) PyLong_AsUnsignedLongMask(PyObject *);
 
+PyAPI_FUNC(int) PyLong_IsPositive(PyObject *);
+PyAPI_FUNC(int) PyLong_IsNegative(PyObject *);
+PyAPI_FUNC(int) PyLong_IsZero(PyObject *);
```

Comment:
I think we should place this in Include/cpython/longobject.h, unless these functions are part of the Limited API.

Reply:
Maybe not. They are just inline functions defined in the header.

Reply:
This header file is for Limited API.

Reply:
Ohh, It seems I may have misunderstood slightly. What I meant is that if these functions are intended for public use, perhaps we don’t need to move their location. For example, as mentioned in the discussion about making them public.

Reply:
Not all public functions are part of the Limited API.  https://docs.python.org/3.13/c-api/stable.html

Reply:
I read the discussion about this API again and I think maybe they should align with the location of `PyLong_GetSign`, so. I will make the change :)